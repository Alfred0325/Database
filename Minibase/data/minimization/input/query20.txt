Q(x, y) :- R(x, y), R(3, y), R(x, 3), R(x, 4), R(z, 6), R(u, v)