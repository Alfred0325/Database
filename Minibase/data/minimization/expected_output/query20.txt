Q(x, y) :- S('ADBS', 12, 7), R(x,3)
